src/components/masters/user-program-enrollments/data/tests/reducer.test.js

```diff
@@ -0,0 +1,62 @@
+import programEnrollmentsReducer from '../reducer';
+import {
+  FETCH_USER_PROGRAM_ENROLLMENTS_REQUEST,
+  FETCH_USER_PROGRAM_ENROLLMENTS_SUCCESS,
+  FETCH_USER_PROGRAM_ENROLLMENTS_FAILURE,
+} from '../constants';
+
+
+const initialState = {
+  loading: false,
+  data: null,
+  error: null,
+};
+
+describe('programEnrollments reducer', () => {
+  it('should return the initial state', () => {
+    expect(programEnrollmentsReducer(undefined, {})).toEqual(initialState);
+  });
+
+  it('should handle FETCH_USER_PROGRAM_ENROLLMENTS_REQUEST', () => {
+    const expected = {
+      ...initialState,
+      loading: true,
+      error: null,
+    };
+    expect(programEnrollmentsReducer(undefined, {
+      type: FETCH_USER_PROGRAM_ENROLLMENTS_REQUEST,
+    })).toEqual(expected);
+  });
+
+  it('should handle FETCH_USER_PROGRAM_ENROLLMENTS_SUCCESS', () => {
+    const expected = {
+      loading: false,
+      data: {
+        course_runs: ['some data'],
+      },
+      error: null,
+    };
+    expect(programEnrollmentsReducer(undefined, {
+      type: FETCH_USER_PROGRAM_ENROLLMENTS_SUCCESS,
+      payload: {
+        data: {
+          course_runs: ['some data'],
+        },
+      },
+    })).toEqual(expected);
+  });
+
+  it('should handle FETCH_USER_PROGRAM_ENROLLMENTS_FAILURE', () => {
+    const expected = {
+      ...initialState,
+      loading: false,
+      error: 'sorry there was an error!',
```

Comment:
Similar to your failure test above, let's expect an actual `Error` object to be returned here #consistency

Reply:
sounds good, I also changed this in the other reducer test as well